action.yml

```diff
@@ -41,3 +38,23 @@ inputs:
   REVIEW_BOT_USER_TOKEN:
     description: 'Token used for authorization when sending the reviews to the LMS'
     required: true
+runs:
+  using: 'composite'
```

Comment:
Question: Do we need the composite action 🤔; I mean can't we work with the docker action itself since we are reading the workflow file; From my understanding it the file will be accessible inside the docker build;

Reply:
Yes

Reverted to Docker based action